src/tools/gophish_cleaner.py

```diff
@@ -32,7 +32,10 @@
 
 # Third-Party Libraries
 from docopt import docopt
-import requests.packages.urllib3
+
+# No type stubs exist for requests.packages.urllib3, so we add "type: ignore"
+# to tell mypy to ignore this library
+import requests.packages.urllib3  # type: ignore
```

Comment:
So this is a more substantial change, and applies to all such imports, but thoughts on switching to just `import urllib3`? It looks like [this exists for backwards compatibility](https://github.com/psf/requests/blob/15585909c3dd3014e4083961c8a404709450151c/requests/packages.py#L12-L28) but based on https://github.com/psf/requests/pull/4067 and https://github.com/python/typeshed/issues/6893#issuecomment-1012511758 this is the same thing as `import urllib3`. If we want to make this fault tolerant (in case `requests` went wild and removed the dependency) we can add [urllib3](https://pypi.org/project/urllib3/) a dependency of the project and manually add `types-urllib3` (it is currently a dependency of `types-requests`) for `mypy`. This allows us to remove the `mypy` ignore just because of import shenanigans in the package.

Reply:
I like it!

Reply:
Good thinking!  Done in https://github.com/cisagov/gophish-tools/pull/123/commits/7a357f006b969026244f60021402b46bc40bf955 - please take a look and let me know if I missed anything.